Client/Frontend/Library/BookmarksPanel.swift

```diff
@@ -197,6 +228,18 @@ class BookmarksPanel: SiteTableViewController, LibraryPanel {
         doDelete()
     }
 
+    func didAddBookmarkNode() {
+        flashLastRowOnNextReload = true
+    }
+
+    func flashRow(at indexPath: IndexPath) {
+        tableView.selectRow(at: indexPath, animated: true, scrollPosition: .middle)
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(BookmarksPanelUX.RowFlashDelayInMilliseconds)) {
+            self.tableView.deselectRow(at: indexPath, animated: true)
```

Comment:
again, it would be good to have a `if rowIsValid(indexpath) { }` wrapping this for safety.